### PR TITLE
Cb-12098 (iOS) update supportedinterfaceorientations return type

### PR DESCRIPTION
**Pull request by sterlingann**
### Platforms affected
iOS

### What does this PR do?
Updates return type to UIInterfaceOrientationMask in iOS 9+

### What testing has been done on this change?
created test project, automated NPM test

### Checklist
- [x] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [x] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [x] Added automated test coverage as appropriate for this change.

### CODE REVIEW

File: CordovaLib/Classes/Public/CDVViewController.m [hunk 1]
```diff
@@ -419,7 +419,13 @@ - (BOOL)shouldAutorotate
...
 }
 
-- (NSUInteger)supportedInterfaceOrientations
+// CB-12098
+#if __IPHONE_OS_VERSION_MAX_ALLOWED < 90000  
+- (NSUInteger)supportedInterfaceOrientations  
+#else  
+- (UIInterfaceOrientationMask)supportedInterfaceOrientations
+#endif
+
```
Comment (shazron) [inline CordovaLib/Classes/Public/CDVViewController.m]: remove this empty line

File: bin/templates/project/__PROJECT_NAME__/Classes/MainViewController.m [hunk 2]
```diff
@@ -92,7 +92,13 @@ - (UIWebView*) newCordovaViewWithFrame:(CGRect)bounds
...
 }
 
-- (NSUInteger)supportedInterfaceOrientations 
+// CB-12098
+#if __IPHONE_OS_VERSION_MAX_ALLOWED < 90000  
+- (NSUInteger)supportedInterfaceOrientations
+#else  
+- (UIInterfaceOrientationMask)supportedInterfaceOrientations
+#endif
+
```
Comment (shazron) [inline bin/templates/project/__PROJECT_NAME__/Classes/MainViewController.m]: remove this empty line

**Comment by shazron** [comment]
@sterlingann please update the code according to recommendations in the code review, thanks.

**Comment by sterlingann** [comment]
@shazron Lines are removed, sorry for the delay

**Comment by shazron** [comment]
@sterlingann seems to be merge conflicts, you might need to rebase

**Pull request closed without merging**